src/main/java/uk/co/caprica/vlcjinfo/Section.java

```diff
@@ -60,6 +60,15 @@ public String value(String key) {
         return values.get(key);
     }
 
+    /**
+     * Get all values fr a Section
```

Comment:
Typo in "for".
"Section" should not be capitalised.
Sentence here should end with full-stop.